src/components/form/CreateNew.jsx

```diff
@@ -1,17 +1,22 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { useDispatch } from '../../hooks/markDownContext';
 import { createMarkDown } from '../../actions/markDownActions';
 
 export default function CreateNew()  {
   const [title, setTitle] = useState('');
-  const dispatch = useDispatch(); 
+  const [id, setId] = useState('');
+  const dispatch = useDispatch();
 
   const handleSubmit = (event) => {
     event.preventDefault();
-    dispatch(createMarkDown(title));
+    dispatch(createMarkDown({ id, title }));
     setTitle('');
   };
 
+  useEffect(() => {
```

Comment:
you probably dont need this. you can create this id at the time of dispatch on line 12:

```js
dispatch (createMarkDown({ id: Date.now().....
```